src/aind_data_schema_models/mouse_anatomy.py

```diff
@@ -0,0 +1,45 @@
+"""Module for Mouse Anatomy"""
+
+from importlib_resources import files
+from pydantic import BaseModel, ConfigDict, Field
+
+from aind_data_schema_models.registries import RegistryModel, map_registry
+from aind_data_schema_models.utils import create_literal_class, read_csv, one_of_instance
+
+
+class MouseAnatomyModel(BaseModel):
+    """Base model config"""
+
+    model_config = ConfigDict(frozen=True)
+
+    id: int = Field(..., title="Structure EMAPA ID")
```

Comment:
this is redundant with the registry_identifier below.

Reply:
I had this like this so that id would be an int and people wouldn't have to do string comprehension to pull it out.